docs/GlobalAPI.md

```diff
@@ -243,6 +243,7 @@ Use `describe.each` if you keep duplicating the same test suites with different
     - `%o` - Object.
     - `%%` - single percent sign ('%'). This does not consume an argument.
 - `fn`: `Function` the suite of tests to be ran, this is the function that will receive the parameters in each row as function arguments.
+- Optionally, you can provide a `timeout` (in milliseconds) for specifying how long to wait before aborting. _Note: The default timeout is 5 seconds._
```

Comment:
👋 Thanks for proactively addressing #6749 before I even wrote it!

I think it is worth making it clear that this timeout is not for the entire parameterized test / test suite but for the individual "row" of data. For example:

> Optionally, you can provide a `timeout` (in milliseconds) for specifying how long to wait **for each row** before aborting. _Note: The default timeout is 5 seconds._

Reply:
@mattphillips mind addressing that?